src/xcode/ENA/ENA/Source/Services/CoronaTestService/FamilyMemberPCRTest.swift

```diff
@@ -0,0 +1,74 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+struct FamilyMemberPCRTest: Equatable, Hashable {
+
+	// MARK: - Internal
+
+	var displayName: String
+
+	var registrationDate: Date
+	var registrationToken: String?
+	var qrCodeHash: String
+	var isNew: Bool
+
+	var testResult: TestResult
+	var finalTestResultReceivedDate: Date?
+	var testResultWasShown: Bool
+	
+	var certificateSupportedByPointOfCare: Bool
+	var certificateConsentGiven: Bool
+	var certificateRequested: Bool
+	
+	var uniqueCertificateIdentifier: String?
+
+	var isLoading: Bool
+
+}
+
+extension FamilyMemberPCRTest: Codable {
```

Comment:
Same question again, why implemented `Codable` in an extension?

Reply:
Same answer as above :)